osgp/platform/osgp-secret-management/src/main/java/org/opensmartgridplatform/secretmanagement/application/endpoints/SecretManagementEndpoint.java

```diff
@@ -130,8 +117,7 @@ private String getSecretsRequestToString(GetSecretsRequest request) {
             marshaller.marshal(request, baos);
 
             return baos.toString();
-        }
-        catch(JAXBException e) {
+        } catch (JAXBException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1166)

Reply:
Looks like a false positive